ui/hooks/useTransactionInsights.js

```diff
@@ -40,8 +38,7 @@ const useTransactionInsights = ({ txData }) => {
   );
 
   const { txParams, chainId, origin } = txData;
-  const networkId = CHAIN_ID_TO_NETWORK_ID_MAP[chainId];
-  const caip2ChainId = `eip155:${networkId ?? stripHexPrefix(chainId)}`;
+  const caip2ChainId = `eip155:${parseInt(chainId, 16).toString()}`;
```

Comment:
Do we need to be worried about `parseInt` returning NaN?

Reply:
seems like this question has been [addressed in a separate thread?](https://github.com/MetaMask/metamask-extension/pull/20652#discussion_r1349304037)